### PR TITLE
Issue/3

### PR DESCRIPTION
**Pull request by jacobsorme**
All tests added! :100: :1st_place_medal: :fire:

### CODE REVIEW

**Review by erinol — APPROVED** [review]
Seems like a very good PR

**Review by mkforsb — APPROVED** [review]
:+1:

File: tests/piqueserver/test_badmin.py [hunk 1]
```diff
@@ -25,13 +27,17 @@ def test_score_grief2(self):
...
         self.assertEqual(0,badmin.score_grief(connection,player))
 
+    # Give grief score to player - blocks destroyed
     def test_score_grief3(self):
         connection = Mock()
         connection.colors = True
         connection.protocol.players = []
 
         player = Mock()
-        player.blocks_removed = [[110,None]]
```
Comment (NotAFile) [inline tests/piqueserver/test_badmin.py:34]: Please make sure your files follow the [pep8](https://www.python.org/dev/peps/pep-0008/) style in the future. You can use tools such as `flake8` to ensure they conform or even use `autopep8` to format it automatically.

File: tests/piqueserver/test_badmin.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 
 class TestBadmin(unittest.TestCase):
+    # Requires time to be positive
     def test_score_grief1(self):
```
Comment (NotAFile) [inline tests/piqueserver/test_badmin.py:8]: You should make the name of the test describe it's purpose. For example in this case, `test_score_time_is_positive`. When the test fails, the name of the test will appear. If it has a name that includes numbers, it is not clear which test exactly failed without looking it up in the source file.

File: tests/piqueserver/test_badmin.py [hunk 3]
```diff
@@ -4,6 +4,7 @@
 
 
 class TestBadmin(unittest.TestCase):
+    # Requires time to be positive
```
Comment (NotAFile) [inline tests/piqueserver/test_badmin.py:7]: Comments describing a function belong in docstrings:

```python
def my_func():
    """my super helpful docstring

    it describes the function very well
    """

File: tests/piqueserver/test_badmin.py [hunk 4]
```diff
@@ -41,4 +47,4 @@ def returnZero():
 
         badmin.reactor.seconds = returnZero
```
Comment (NotAFile) [inline tests/piqueserver/test_badmin.py:48]: `reactor.seconds` does not get restored after this. This means other tests might break. I also suggest you use the functionality of `unittest.mock` to set return values and `patch` to return `reactor.seconds` to the original value (https://docs.python.org/3/library/unittest.mock.html)

File: tests/piqueserver/test_core_commands.py [hunk 5]
```diff
@@ -2,9 +2,12 @@
 from unittest.mock import Mock
 
 from piqueserver import core_commands
+import piqueserver.core_commands.movement as Movement
```
Comment (NotAFile) [inline tests/piqueserver/test_core_commands.py:5]: `Movement` is not pep8 compliant here. Module names are required to be all lower case. UpperCamelCase signifies a class.

**Review by NotAFile — COMMENTED** [review]
Another thing here is that the location of these files is not correct. The structure of `tests/` mimics the structure of piqueserver itself. Hence, these tests belong in `tests/piqueserver/scripts/`

**Pull request merged**